### PR TITLE
Watt standalone

### PR DESCRIPTION
**Pull request by mimse**
Updated watt components to be standalone

## References

- https://github.com/Energinet-DataHub/greenforce-frontend/issues/1744

### CODE REVIEW

File: libs/ui-watt/src/lib/components/clipboard/watt-copy-to-clipboard.directive.spec.ts [hunk 1]
```diff
@@ -17,13 +17,13 @@
...
 import userEvent from '@testing-library/user-event';
 
-import { WattToastModule } from '../toast';
+import { WattToastComponent } from '../toast';
 import { WattCopyToClipboardDirective } from './watt-copy-to-clipboard.directive';
 
 describe(WattCopyToClipboardDirective.name, () => {
   it('shows toast on click', async () => {
     await render(`<span wattCopyToClipboard>Text</span>`, {
-      imports: [WattToastModule.forRoot(), WattCopyToClipboardDirective],
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/components/clipboard/watt-copy-to-clipboard.directive.spec.ts:26]: suggestion: a bit weird that this works. Probably it's because `WattToastComponent` imports `MatLegacySnackBarModule`, which in turn provides the necessary providers. Instead of importing `WattToastComponent`, which looks unrelated to this directive anyway, let's use the `importProvidersFrom(MatLegacySnackBarModule)` pattern.

**Review by dzhavat — COMMENTED** [review]
praise: fantastic job. Looks much better without NgModules. 🚀

Did you find it easy to convert SCAMs to Standalone?

**Comment by mimse** [comment]
Most of them were easy to convert, the ones with forRoot were a bit harder :-)

**Review by dzhavat — APPROVED** [review]
